locations/templates/locations/location_page.html

```diff
@@ -1,68 +1,241 @@
 {% extends "base.html" %}
 
-{% load wagtailcore_tags wagtailimages_tags %}
+{% load wagtailcore_tags wagtailimages_tags lazyimages_tags %}
 
 {% block body_class %}template-locationpage{% endblock %}
 
 {% block content %}
 
-    <div class="row">
-        <div class="col">
-            <div class="text-center">
-                <h1>{{ page.title }}</h1>
-                <div class="intro">
-                    {{ page.intro }}
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.5.0/css/ol.css" type="text/css">
```

Comment:
What are these imports for ? maps?

Use an integrity hash if you are going to import from a 3rd party site (https://www.srihash.org/)